services/QuillLMS/engines/evidence/lib/evidence/evidence/plagiarism_check.rb

```diff
@@ -158,7 +154,10 @@ def feedback_object
       # we know that there's a minimum number of words that have to be identical.  This function
       # confirms that minimum amount of overlap to justify doing a set of Levenshtein calculations.
       source_arrays.any? do |source_array|
-        (source_array & target_array).length >= (MATCH_MINIMUM - FUZZY_CHARACTER_THRESHOLD)
+        [
+         (target_array - source_array).length,
```

Comment:
This looks good. The only other slight optimization I might suggest is that you can early return the first is above the threshold, i.e.
```ruby
(target_array - source_array).length <= FUZZY_CHARACTER_THRESHOLD 
|| (source_array - target_array).length <= FUZZY_CHARACTER_THRESHOLD

Reply:
Good catch.  I'll make that update and deploy.